test/inferCNN_cfg.py

```diff
@@ -20,26 +31,31 @@
 datadir = "/uscms_data/d3/ssekhar/CMSSW_11_1_2/src/TrackerStuff/PixelHitsCNN/data"
 
 # setup minimal options
-options = VarParsing("python")
-options.setDefault("inputFiles", 'root://cms-xrd-global.cern.ch//store/data/Run2018D/SingleMuon/ALCARECO/SiPixelCalSingleMuon-ForPixelALCARECO_UL2018-v1/20000/A28530AF-8EB6-814B-9645-642058515DA2.root')  # noqa
-options.parseArguments()
+# options = VarParsing("python")
+# options.setDefault("inputFiles", 'root://cms-xrd-global.cern.ch//store/data/Run2018D/SingleMuon/ALCARECO/SiPixelCalSingleMuon-ForPixelALCARECO_UL2018-v1/20000/A28530AF-8EB6-814B-9645-642058515DA2.root')  # noqa
+# options.parseArguments()
 
 # define the process to run
 process = cms.Process("TEST")
 
 # minimal configuration
 process.load("FWCore.MessageService.MessageLogger_cfi")
 process.MessageLogger.cerr.FwkReport.reportEvery = 1
-process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(10))
+process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(-1))
 process.source = cms.Source("PoolSource",
-    fileNames=cms.untracked.vstring(options.inputFiles))
+    fileNames=cms.untracked.vstring("root://cms-xrd-global.cern.ch//store/data/Run2018C/SingleMuon/RAW/v1/000/320/065/00000/8C070B38-338E-E811-A4D1-FA163E781D28.root")
 
 # process options
 process.options = cms.untracked.PSet(
     allowUnscheduled=cms.untracked.bool(True),
     wantSummary=cms.untracked.bool(True),
 )
 
+# to get the conditions you need a GT
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '112X_dataRun2_v7', '')
```

Comment:
this GT will work in 11_2_X CMSSWs for sure (might work in other ones too), let me know if you need a different version of the GT